app/views/nsm/additional_fees/_adjusted.html.erb

```diff
@@ -1,6 +1,30 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <p><%= t('.no_data') %></p>
+    <% if records.rows.none? { _1.any_adjustments? } %>
+      <p><%= t('.no_data') %></p>
+    <% else %>
+      <%=
+      caption = t('.table_caption')
+      head = Nsm::V1::AdditionalFee.headers
+      rows = records.rows.reject{|item| item.type == :total}.map do |additional_fee|
```

Comment:
nit: can use _1 shorthand notation here if you want: 
record.rows.reject{ _1.type == :total } 

but not too fussed either way